### PR TITLE
[pmdk] Update to 1.8

### PR DESCRIPTION
**Pull request by NancyLi1013**
Update to the latest released version 1.8.

Related issue #9883. 

Remove libvmem from solution Items. Please refer to these [changes](https://github.com/pmem/pmdk/releases).

Note: No feature needs to test.

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

File: ports/pmdk/portfile.cmake [hunk 1]
```diff
@@ -55,4 +55,4 @@ vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/pmdk)
 
 vcpkg_copy_pdbs()
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
```
Comment (strega-nil) [inline ports/pmdk/portfile.cmake]: Could you remove this change?

Comment (NancyLi1013) [reply ports/pmdk/portfile.cmake]: Thanks for your reminder. Have updated.

**Comment by strega-nil** [comment]
Cools, thanks @NancyLi1013 :)

**Pull request merged**